api/proto/common/v2/common_v2.proto

```diff
@@ -36,12 +34,14 @@ message BlobHeader {
 message BlobCertificate {
   // blob_header contains data about the blob.
   BlobHeader blob_header = 1;
-  // relays is the list of relays that are in custody of the blob.
+  // signature over keccak hash of the blob_header that can be verified by blob_header.payment_header.account_id
+  bytes signature = 2;
+  // relay_keys is the list of relay keys that are in custody of the blob.
   // The relays custodying the data are chosen by the Disperser to which the DisperseBlob request was submitted.
   // It needs to contain at least 1 relay number.
   // To retrieve a blob from the relay, one can find that relay's URL in the EigenDARelayRegistry contract:
   // https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/core/EigenDARelayRegistry.sol
-  repeated uint32 relays = 2;
+  repeated uint32 relay_keys = 3;
```

Comment:
it's fine to settle on "relay key" instead of "relay ID", but can you get all the "relay ID"s renamed to make it consistent, eg: https://github.com/Layr-Labs/eigenda/blob/9bc28688fbf5dbc11cdc0096549a0d34f75ab19e/relay/server.go#L72